docs/connect.md

```diff
@@ -0,0 +1,84 @@
+# Plumbery
+
+Note: “Plumbery” is a code name used during the development of this project and will change in the future.
+
+This file documents the exports of the library.
+
+## connect(location, config)
+
+Connects and returns an `Organization` for `location`.
+
+| Name               | Type                                               | Description                                                                                                                        |
+| ------------------ | -------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| `location`         | `String`                                           | The Ethereum address or ENS domain of an Aragon organization.                                                                      |
+| `config`           | `Object`                                           | The optional configuration object.                                                                                                 |
+| `config.connector` | `Connector \| [String, Object] \| String`          | Accepts a `Connector` instance, and either a string or a tuple for embedded connectors and their config. Defaults to `"thegraph"`. |
+| `config.ipfs`      | `(ipfsIdentifier: String, path: String) => String` | A function that resolves an IPFS identifier and a path into an HTTP URL. Defaults to gateway.ipfs.io.                              |
+| returns            | `Promise<Organization>`                            | An `Organization` instance.                                                                                                        |
+
+### Errors
+
+| Type                   | Description                                            |
+| ---------------------- | ------------------------------------------------------ |
+| `ConnectionError`      | Gets thrown if the connection fails.                   |
+| `OrganizationNotFound` | Gets thrown if the organization doesn’t seem to exist. |
+
+### Example
+
+```js
+import { connect } from 'plumbery'
+
+// Connections should get wrapped in a try / catch to capture connection errors.
+try {
+  // Connect to an org using the default config
+  const org1 = await connect('org1.aragonid.eth')
+
+  // Specify a connector
+  const org2 = await connect('org2.aragonid.eth', { connector: 'thegraph' })
+
+  // Specify a connector and its config
+  const org3 = await connect('org3.aragonid.eth', {
+    connector: ['thegraph', { daoSubgraphUrl: 'http://…' }],
+  })
+
+  // Custom connector
+  const org4 = await connect('org4.aragonid.eth', {
+    connector: new CustomConnector(),
```

Comment:
Maybe we could add the`IConnector` the custom connector needs to implement.

Reply:
Yes, I try to use JS only in the code examples but we could add it as a comment?